avocado/utils/process.py

```diff
@@ -433,6 +434,7 @@ def _fill_results(self, rc):
         if self.verbose:
             log.info("Command '%s' finished with %s after %ss", self.cmd, rc,
                      self.result.duration)
+        self.result.pid = self._popen.pid
```

Comment:
Since all other `CmdResult` attributes are just that, attributes, this matches the code style.  (The other side of this comment is that `SubProcess` contains `get_pid()`, which is a method and not an attribute, but that is fine).